mobile/src/screens/profile-settings/profile-settings.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import LinearGradient from 'react-native-linear-gradient';
+
+import { Header, ScrollView, Switch, View } from '#libs/components/components';
+import { AppColor } from '#libs/enums/enums';
+import { useAppForm } from '#libs/hooks/hooks';
+import { type NavigationScreenProperties } from '#libs/types/types';
+
+import { Setting } from './components/components';
+import { DEFAULT_NOTIFICATION_SETTINGS_VALUES } from './libs/constants';
+import { styles } from './styles';
+
+const ProfileSettings = ({
+  navigation,
+  route,
+}: NavigationScreenProperties): JSX.Element => {
+  const { control } = useAppForm({
+    defaultValues: DEFAULT_NOTIFICATION_SETTINGS_VALUES,
+  });
+
+  return (
+    <LinearGradient
+      colors={[AppColor.WHITE, AppColor.BLUE_100]}
+      style={styles.linearGradient}
+    >
+      <Header navigation={navigation} title={route.name} isArrowVisible />
```

Comment:
In Header component, let's use navigation from hook (`const navigation = useNavigation()` and make title property optional, if it is not passed, use `const { name } = useAppRoute();`